packages/legend-graph/src/models/protocols/pure/v1/V1_Core_SystemModels.json

```diff
@@ -19,6 +19,13 @@
       "name": "doc",
       "tags": ["doc", "todo"]
     },
+    {
+      "stereotypes": ["bitemporal", "businesstemporal", "processingtemporal"],
```

Comment:
TODO: We should expose the `temporal` profile in the dropdown, see `getExtraExposedSystemElementPath()`